nncf/layers.py

```diff
@@ -47,9 +50,36 @@ def from_module(module):
         return nncf_conv
 
 
+NNCF_PADDING_VALUE_ATTR_NAME = 'nncf_padding_value'
+OPTIONAL_PARAMETERS_REGISTRY.register(NNCF_PADDING_VALUE_ATTR_NAME)
+
+
 class NNCFConv2d(_NNCFModuleMixin, nn.Conv2d):
     op_func_name = "conv2d"
 
+    def __init__(
+        self,
+        in_channels: int,
+        out_channels: int,
+        kernel_size,
+        stride=1,
+        padding=0,
+        dilation=1,
+        groups: int = 1,
+        bias: bool = True,
+        padding_mode: str = 'zeros'
+    ):
+        super().__init__(in_channels, out_channels, kernel_size, stride, padding, dilation, groups,
+                         bias, padding_mode)
+        self._set_padding_value(Parameter(torch.zeros([1]), requires_grad=False))
+        self.custom_forward_fn_ = self.custom_forward.__func__
```

Comment:
You are actually setting an instance attribute here. The rest of the code seems to be written as if the `custom_forward_fn` should be a class attribute. I propose keeping it a class attribute, if this does not break anything, i.e. just do `def custom_forward_fn_` in the NNCF... classes. Perhaps should also rename `custom_forward_fn_` to `_custom_forward_fn`.

Reply:
Done